packages/ove-app-html/docs/UTIL_BACKGROUND.md

```diff
@@ -0,0 +1,17 @@
+# Background Utility
+
+If a web page or an app has a transparent background by design, if required a background colour of choice can be set using this utility:
```

Comment:
If a web page or an app has a transparent background by design, ~~if required~~ a background colour of choice can be set using this utility